posthog/api/error_tracking.py

```diff
@@ -115,10 +115,11 @@ def retrieve(self, request, *args, **kwargs):
             record = fingerprint_queryset.filter(fingerprint=fingerprint).first()
 
             if record:
-                if not record.issue_id == self.request.GET.get("pk"):
+                if not str(record.issue_id) == self.kwargs.get("pk"):
```

Comment:
the issue was this was always evaluating as `False` when a fingerprint param was included